pkg/network/event_common.go

```diff
@@ -214,8 +249,9 @@ type ConnectionStats struct {
 	IPTranslation *IPTranslation
 	Via           *Via
 
-	Monotonic StatCounters
-	Last      StatCounters
+	Monotonic StatCountersByCookie
+
+	Last StatCounters
```

Comment:
Maybe add a small comment here explaining the the use of Connection cookies and the problem we're addressing?